acceptance-tests/lib/cmd.ts

```diff
@@ -49,6 +51,7 @@ export function createProcess(
       GITHUB_TOKEN: config.githubToken,
     },
     stdio: [null, null, null, 'ipc'], // This enables interprocess communication (IPC)
+    cwd: path.join(process.cwd(), config.testDir),
```

Comment:
This sets all of the commands we execute to run in the test directory.  So files generated should be placed within there.